### PR TITLE
Adding Spacescape by DevKage.

### PR DESCRIPTION
**Pull request by ufrshubham**

### CODE REVIEW

**Review by wolfenrain — APPROVED** [review]

**Review by spydon — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -143,6 +149,9 @@ By Tommy Buonomo ([Frenchie Games](https://frenchiegames.app/#/))
 <!--lint ignore double-link-->
 - Dino Run - [Android](https://play.google.com/store/apps/details?id=com.nobs.dino_run) - Dino Run is casual 2D infinite side scroller game, where you play as a cute little dino. All you have to do is avoid getting hit by enemies too many times. The longer you can do that, higher your score goes. By [DevKage](https://www.youtube.com/devkage)
 
+<!--lint ignore double-link-->
+- Spacescape - [Android](https://play.google.com/store/apps/details?id=com.nobs.spacescape) - Spacescape is a simple and minimalistic 2D space shooter game made using the awesome Flame engine. Just shoot all the enemies coming your way to achieve the highest score. By [DevKage](https://www.youtube.com/devkage)
```
Comment (luanpotter) [inline README.md:153]: I think we can combine these entries? the `App Releases` section was to allow non-open-source games to appear as well, if your game is open source it should be on the feature open source list and you can add both links

Comment (ufrshubham) [reply README.md:153]: Sorry, but I didn't understand what exactly you want me to do. 😅 
- Should I completely remove `Dino Run` and `Spacescape` from `App Releases`? 
- Or should I merge them into single entry? If so, should it be in any specific format?
- Or should I add Play Store links to corresponding entries under `Open Source`?

Comment (luanpotter) [reply README.md:153]: sorry if it wasn't clear! I meant:

* you can definitely have separated entries for each game
* we should have only one entry on Open Source (if your game is open source) OR an entry on the non-open-source section (else). not both
* you can have github, store link, or any other links on the Open Source section

At least that is what I remembered we discussed about the non-open-source section years ago haha.
But I leave this for @erickzanardo and @impulse to confirm or deny.

Comment (ufrshubham) [reply README.md:153]: Okay, I think I get it now. I'll wait for others to reply. Maybe the `CONTRIBUTING.md` also needs some rework then.

Comment (ufrshubham) [reply README.md]: Removed `Dino Run` from `App Releases` section and added play store links under `Open Source` section.

File: README.md [hunk 2]
```diff
@@ -98,15 +102,18 @@
...
+
+<!--lint ignore double-link-->
+- [Spacescape](https://github.com/ufrshubham/spacescape) - A 2D top-down space shooter made using Flutter and Flame engine. Get it on [Play Store](https://play.google.com/store/apps/details?id=com.nobs.spacescape). By [DevKage](https://www.youtube.com/devkage)
 
 #### Endless Runner
 
 <!--lint ignore double-link-->
 - [BGUG](https://github.com/fireslime/bgug) - Break Guns Using Gems is a fast paced side-scrolling platformer with innovative controls and and a gun-related twist. By [FireSlime](https://fireslime.xyz)
 
 <!--lint ignore double-link-->
```
Comment (luanpotter) [inline README.md:115]: I think now you can remove this linter exception right? since now it's a single link

File: README.md [hunk 3]
```diff
@@ -12,7 +12,8 @@
       <img alt="License: CC0" src="https://img.shields.io/badge/license-CC0-lightgray?style=flat-square" />
     </a>
     <a href="https://flame-engine.org">
-      <img alt="Powered by Flame" src="https://img.shields.io/badge/Powered%20by-%F0%9F%94%A5-orange.svg?style=flat-square" />
+      <img alt="Powered by Flame" src="https://img
+.shields.io/badge/Powered%20by-%F0%9F%94%A5-orange.svg?style=flat-square" />
```
Comment (luanpotter) [inline README.md:16]: is this change needed?

**Review by luanpotter — APPROVED** [review]
LGTM with 2 final comments
